iceoryx_hoofs/design/include/iox/newtype.hpp

```diff
@@ -14,24 +14,20 @@
 // limitations under the License.
 //
 // SPDX-License-Identifier: Apache-2.0
-#ifndef IOX_HOOFS_CXX_NEWTYPE_HPP
-#define IOX_HOOFS_CXX_NEWTYPE_HPP
-
+#ifndef IOX_HOOFS_DESIGN_NEWTYPE_HPP
+#define IOX_HOOFS_DESIGN_NEWTYPE_HPP
+
+#include "detail/newtype/assignment.hpp"
+#include "detail/newtype/comparable.hpp"
+#include "detail/newtype/constructor.hpp"
+#include "detail/newtype/convertable.hpp"
+#include "detail/newtype/internal.hpp"
+#include "detail/newtype/protected_constructor.hpp"
+#include "detail/newtype/sortable.hpp"
```

Comment:
Why not using the absolute paths?

```suggestion
#include "iox/detail/newtype/assignment.hpp"
#include "iox/detail/newtype/comparable.hpp"
#include "iox/detail/newtype/constructor.hpp"
#include "iox/detail/newtype/convertable.hpp"
#include "iox/detail/newtype/internal.hpp"
#include "iox/detail/newtype/protected_constructor.hpp"
#include "iox/detail/newtype/sortable.hpp"
```